### DISCUSSION TITLE
Flux with local HTTP gitea server, issues with auth

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [fluxcd/flux2#4972] [Q&A]
```
flux bootstrap gitea   --token-auth   --owner=gitea   --repository=demo-repo   --branch=master   --path=flux   --personal --hostname http://gitea.local:3000
► connecting to http://gitea.local:3000
► cloning branch "master" from Git repository "http://gitea.local:3000/gitea/demo-repo.git"
⚠  clone failure: basic auth cannot be sent over HTTP
⚠  clone failure: basic auth cannot be sent over HTTP
✗ failed to clone repository: basic auth cannot be sent over HTTP
```

Any way around this? Or do I have to setup some kind of a reverse proxy? I am just playing around with Flux for the first time but I cannot get it to work

Seems that `allow-insecure-http` is not supported for gitea bootstrap command

**Comment by irecloud** [comment]
Hi Nikola
You can try with this. Have a repo created on gitea as an example below, I have a repo "admin" under a user "admin"

` flux bootstrap git --url=http://gitea.example.com/admin/admin.git --username=***** --password=*****  --allow-insecure-http=true --token-auth=true --branch=main  --namespace=shahid-flux-deploy --interval=30m`
